elixir-europe-training/Training-Metrics-Database
# Clean up dev experience

This PR tries to clean up the dev environment in order to make it easier to get into the project.

It will:
- Update the README to reflect the current reality
- Add some utility .gitignore entries
- Remove unnecessary .DS_Store
- Make metabase optional in order to reduce complexity in dev scenarios
- Update the example data set in order to make it work out of the box

To test:
- Follow the README instructions
- Expect the system to start and load data without problems
- Try using the gitpod (I am not familiar with how to test this so feel free to suggest changes)
